web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :screen_name, :string
+    field :profile_image_url, :string
+    field :twitter_id, :integer
+    field :name, :string
```

Comment:
my plan right now is to just stick with Twitter. That was the scope of the original game and I'd hate to over optimize this early.